static_timing/build_defs.bzl

```diff
@@ -41,13 +42,14 @@ def _run_opensta_impl(ctx):
         "NETLIST": netlist.path,
         "TOP": synth_info.top_module,
         "LOGFILE": sta_log.path,
-        "LIBERTY": liberty_file.path,
-        "TCL_LIBRARY": opensta_runfiles_dir + "/tk_tcl/library",
+        "LIBERTY": default_liberty_file.path,
+        "ADDITIONAL_LIBERTIES": ",".join([f.path for f in additional_liberty_files]),
+        "TCL_LIBRARY": opensta_runfiles_dir + "/google3/third_party/tcl_tk/libs/library",
```

Comment:
This shouldn't be google3. Revert it to the old value.

Reply:
Oops, thanks for the catch. Need to be more careful on using the patch